go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/go-telegram-bot-api/telegram-bot-api/v5
+module github.com/sphantix/telegram-bot-api
```

Comment:
Needs to fix that before merge.

Reply:
@albertosouza i fixed it (https://github.com/go-telegram-bot-api/telegram-bot-api/pull/737)